x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/stats.tsx

```diff
@@ -104,17 +105,17 @@ export function MobileStats({
 
   const metrics: MetricDatum[] = [
     {
-      color: euiTheme.eui.euiColorDisabled,
+      color: euiTheme.eui.euiColorLightestShade,
       title: i18n.translate('xpack.apm.mobile.metrics.crash.rate', {
-        defaultMessage: 'Crash Rate (Crash per minute)',
-      }),
-      subtitle: i18n.translate('xpack.apm.mobile.coming.soon', {
-        defaultMessage: 'Coming Soon',
+        defaultMessage: 'Crashes',
       }),
       icon: getIcon('bug'),
-      value: 'N/A',
-      valueFormatter: (value: number) => valueFormatter(value),
-      trend: [],
+      value:
+        data?.currentPeriod?.crashes?.value /
+          data?.currentPeriod?.sessions?.value ?? NOT_AVAILABLE_LABEL,
+      valueFormatter: (value: number) =>
+        valueFormatter(Number((value * 100).toPrecision(2)), '%'),
+      trend: data?.currentPeriod?.crashes?.timeseries,
```

Comment:
The comparison with previous period is missing, is a plan to add it? For some reason, it disappeared also from the other widgets @achyutjhunjhunwala, it's been a change related to it from the charts team?
```suggestion
     extra: getComparisonValueFormatter(data?.previousPeriod.crashes?.value)
```

Reply:
Yes, that might be, but there's also a bug with the widget charts https://github.com/elastic/kibana/issues/152743
I think you should add the comparison, when the bug it's fixed it will display the previous value for the crash rate
c.c @boriskirov @akhileshpok